src/Modulo2.jl

```diff
@@ -663,6 +679,16 @@ function copyto!(a::ZZ2Array{N}, bc::Broadcasted{ZZ2ArrayStyle{N}, <:Any, typeof
     a
 end
 
+#
+# Utility functions
+#
+
+count_ones(a::ZZ2) = ifelse(a.m, 1, 0)
+count_zero(a::ZZ2) = ifelse(a.m, 0, 1)
+# TODO: use views to not reduce over the padding to multiples of 256?
+count_ones(arr::ZZ2Array) = mapreduce(count_ones, +, arr.data; init = 0)
```

Comment:
```suggestion
count_ones(arr::ZZ2Array) = sum(count_ones, arr.data; init = 0)
```
This is more idiomatic and as fast as boiling it down to `mapreduce`.